src/faction.c

```diff
@@ -452,22 +452,22 @@ void faction_addEnemy( int f, int o)
    if (faction_isFaction(f))
       ff = &faction_stack[f];
    else { /* f is invalid */
-      WARN(_("addEnemy: %d is an invalid faction"), f);
+      WARN(_("Faction id '%d' is invalid."), f);
       return;
    }
 
    if (!faction_isFaction(o)) { /* o is invalid */
-      WARN(_("addEnemy: %d is an invalid faction"), o);
+      WARN(_("Faction id '%d' is invalid."), o);
       return;
    }
 
    /* player cannot be made an enemy this way */
    if (f==FACTION_PLAYER) {
-      WARN(_("addEnemy: %d is the player faction"), f);
+      WARN(_("Faction id '%d' is invalid."), f);
```

Comment:
This should still be warning that it's the player faction, shouldn't it?

🕵️

Reply:
Oh no, good catch, thanks! Fixing now.